### PR TITLE
Case for DC-DC converter

### PR DESCRIPTION
**Pull request by cm0x4D**
- [x] Add case item
  - [x] Item (code)
  - [x] Icon
  - [x] Recipe
- [x] Add case item slot to DC/DC converter GUI
- [x] Adapt rendering code to use case only in the presence of the item
- [x] Add animation for casing door.

### CODE REVIEW

File: src/main/java/mods/eln/Eln.java [hunk 1]
```diff
@@ -6645,6 +6647,12 @@ void recipeMiscItem() {
             Character.valueOf('g'), new ItemStack(Blocks.glass_pane),
             Character.valueOf('c'), new ItemStack(Items.dye, 1, 1));
 
+        addRecipe(findItemStack("Casing"),
+            "ppp",
```
Comment (Baughn) [inline src/main/java/mods/eln/Eln.java]: This is a bit expensive for what's a cosmetic item. Maybe make the recipe output 4 or 8 items?

File: src/main/java/mods/eln/Eln.java [hunk 2]
```diff
@@ -6645,6 +6647,12 @@ void recipeMiscItem() {
             Character.valueOf('g'), new ItemStack(Blocks.glass_pane),
             Character.valueOf('c'), new ItemStack(Items.dye, 1, 1));
 
+        addRecipe(findItemStack("Casing"),
```
Comment (Baughn) [inline src/main/java/mods/eln/Eln.java]: This is a bit expensive for what's a cosmetic item. Maybe make the recipe output 4 or 8 items?

File: src/main/java/mods/eln/misc/Obj3D.java [hunk 3]
```diff
@@ -518,7 +518,9 @@ public boolean loadFile(final String filePath, boolean reload) {
                 String line;
                 while ((line = bufferedReader.readLine()) != null) {
                     String[] words = line.split(" ");
-                    if (words[0].equals("o")) {
+                    if (words[0].startsWith("#")) {
+                        // # is a comment - ignore line.
```
Comment (Baughn) [inline src/main/java/mods/eln/misc/Obj3D.java]: Unknown verbs are already ignored. I guess we should log a debug warning for those?

**Review by Baughn — APPROVED** [review]

File: src/main/java/mods/eln/Eln.java (hunk 2 above)
Comment (cm0x4D) [reply src/main/java/mods/eln/Eln.java]: Good point. Totally agree.

**Pull request merged**